docs/_static/vdr/v1.yaml

```diff
@@ -77,13 +77,39 @@ paths:
               $ref: '#/components/schemas/DIDUpdateRequest'
       responses:
         "200":
-          description: DID Document has been updated.
+          description: DID document has been updated.
           content:
             application/json+did-document:
               schema:
                  $ref: '#/components/schemas/DIDDocument'
         "400":
-          description: DID couldn't be updated because the input conflicts or is malformed.
+          description: DID document couldn't be updated because the did param was malformed or the document was deactivated.
+          content:
+            text/plain:
+              schema:
+                type: string
+        "404":
+          description: Corresponding DID document could not be found.
+          content:
+            text/plain:
+              schema:
+                type: string
+        "500":
+          description: An error occurred while processing the request.
+          content:
+            text/plain:
+              schema:
+                type: string
+    delete:
+      summary: Deactivates a Nuts DID document according to the specification.
+      operationId: "deactivateDID"
+      tags:
+        - DID
+      responses:
+        "200":
```

Comment:
Without any return body, a 202 is more suitable.

Reply:
204, no content?